README.md

```diff
@@ -0,0 +1,169 @@
+ <a name="readme-top"></a>
+
+<div align="center">
+  <h1><b>Motorbikes Booking</b></h1>
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [Motorbikes Booking ] <a name="about-project"></a>
+
+The Backend for Motorbikes Booking app. Built using Ruby on Rails.
+
+# Database Schema <a name="database-schema"></a>
+<div align="center">
+ <img src ="Motorbikes.png">
+</div>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Link to [Front-End Project](https://github.com/SaneMyburg/motorbike-front-end)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li>Ruby on Rails </li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Authentication**
+- **CRUD**
+- **Motorbikes Image Upload**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Install Git.
+- Install Ruby
+- Install Ruby on Rails
+- Clone this repo as described in the setup section. 
+
+
+### Prerequisites
+
+In order to run this project you need: To clone or fork and run it in a browser
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/SaneMyburg/motorbike-back-end.git
+```
+
+For more information on how to clone or fork a repository:
+- <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository">How to clone a repo</a>
+- <a href="https://docs.github.com/en/get-started/quickstart/fork-a-repo">How to fork a repo</a>
+
+### Install
+
+- Run ` bundle install `
+- Run ` bundle exec rails db:create db:migrate `
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run ` bundle exec rails server`
+
+### Run tests
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍